### DISCUSSION TITLE
发现附件/图库/image stream/瞬间等使用图片的地方有点不协同的问题

### DISCUSSION THREAD

**Discussion by IshuaWang** [halo-dev/halo#7120] [General]
比如附件只能上传本地的图片，这是合理的。图库添加图片可以从附件里选，也可以直接引用网络的图片，包括使用image stream插件直接引用平台的图片，也可以把平台的图片转存成附件再引用。但是使用瞬间这个插件选择图片的时候却只能从附件库和image stream里选，却查看不到图库里有些图片，具体指的是图库里直接用图片地址链接的那些图片。如果这样的话图库的意义是什么呢？我感觉图库的作用是可以方便地插入非平台的网络图片，但是瞬间却不能引用，这就有点奇怪。瞬间是不是应该可以直接引用网络图片比较好？

**Comment by guqing** [comment]
图库插件的作用是画廊或相册的作用允许在主题端展示，跟瞬间插件没有任何关系，瞬间插件你要添加图片从附件库选择不是很合理吗 你发朋友圈也的从手机相册选择，当然你可以直接粘贴图片到瞬间的内容里面会自动上传到附件库但是主题就没法布局图片了比如九宫格

**Reply by IshuaWang** [reply]
感谢回复啊，虽然具体细节暂时不太懂
